ert_shared/storage/endpoints/responses.py

```diff
@@ -21,7 +21,12 @@ def _calculate_misfit(
     misfit = (difference / observation_std) ** 2
     sign = difference > 0
 
-    return {"value": misfit, "sign": sign, "obs_index": obs_index}
+    return {
+        "value": misfit,
+        "sign": sign,
+        "obs_index": obs_index,
```

Comment:
do we still need to send the observation index? As far as i remember it didn't make much sense in the gui? But i might remember wrong 🤔

Reply:
> do we still need to send the observation index? As far as i remember it didn't make much sense in the gui? But i might remember wrong

Decided to leave it there as we weren't sure :shrug: yesterday if it is allowed to two obs values with the same location; ie. `obs_index` separates them.